pyeight/user.py

```diff
@@ -10,14 +10,15 @@
 from datetime import datetime, timedelta
 import logging
 import statistics
-import time
+
+import pytz
```

Comment:
Can you add this functionality without the pytz library?  I'd like to stay away from bundling in non-standard libraries if possible.

Reply:
as far as I can tell, not unless we set the minimum version of the package to 3.9 which introduced this: https://docs.python.org/3/library/zoneinfo.html

Reply:
FWIW the latest version of HA sets 3.9 to a minimum, so if we are primarily concerned with HA, we can update the minimum. I'm assuming old versions of the package are still available for older Python versions

Reply:
I went ahead and made that change - if you'd like me to revert the last commit just let me know, but this:
- Bumps min version to 3.9
- uses stdlib zoneinfo for timezones